dune.module

```diff
@@ -14,4 +14,4 @@
 Module: dune-xt-functions
 Version: 2.5.0-dev
 Maintainer: dune-xt-dev@listserv.uni-muenster.de
-Depends: dune-common (>= 2.5) dune-testtools (>= 2.5) dune-geometry (>= 2.5) dune-grid (>= 2.5) dune-xt-common (>=2.5) dune-xt-grid (>= 2.5) dune-xt-la (>= 2.5)
+Depends: dune-common (>= 2.5) dune-testtools (>= 2.5) dune-geometry (>= 2.5) dune-grid (>= 2.5) dune-xt-common (>=2.5) dune-xt-grid (>= 2.5) dune-xt-la (>= 2.5) dune-xt-data
```

Comment:
Since dune-xt-data is not a small repo (and will probably grow over time) I would rather have this as an optional dependency, at least at some point...

Reply:
It's the same amount of data though we previously had to download/build via external-libraries (lebedev+matrix exp) and cmake (spe10), only now you only need to download this once at init time of the supermodule, instead of per build-dir / external-libraries config.

Reply:
Sure, and it is a good idea and an improvement over the current state! But in general, I think this is too much as a hard dependency...